modules/core/04-channel/types/msgs.go

```diff
@@ -486,16 +486,15 @@ var _ sdk.Msg = &MsgChannelUpgradeInit{}
 func NewMsgChannelUpgradeInit(
 	portID, channelID string,
 	proposedUpgradeChannel Channel,
-	timeoutHeight clienttypes.Height,
-	timeoutTimestamp uint64,
+	time UpgradeTimeout,
 	signer string,
 ) *MsgChannelUpgradeInit {
 	return &MsgChannelUpgradeInit{
 		PortId:                 portID,
 		ChannelId:              channelID,
 		ProposedUpgradeChannel: proposedUpgradeChannel,
-		TimeoutHeight:          timeoutHeight,
-		TimeoutTimestamp:       timeoutTimestamp,
+		TimeoutHeight:          time.TimeoutHeight,
```

Comment:
thanks for this PR! 

i think the UpgradeTry message also needs to be updated further down this file in a similar way to what you've already done with ChanUpgradeInit